Mathlib/SetTheory/Cardinal/Finite.lean

```diff
@@ -168,13 +168,13 @@ lemma card_preimage_of_injOn {f : α → β} {s : Set β} (hf : (f ⁻¹' s).Inj
 lemma card_preimage_of_injective {f : α → β} {s : Set β} (hf : Injective f) (hsf : s ⊆ range f) :
     Nat.card (f ⁻¹' s) = Nat.card s := card_preimage_of_injOn hf.injOn hsf
 
-@[simp] lemma card_univ : Nat.card (univ : Set α) = Nat.card α :=
+lemma card_univ : Nat.card (univ : Set α) = Nat.card α :=
```

Comment:
For reviewers: This one is now provable by simp, after Data/Set/Card is imported, and so having this as @[simp] creates a problem for the simpNF linter. Fortunately, the only time it's used as a simp lemma is in the immediately following lemma.